api/src/main/java/org/openmrs/api/ObsService.java

```diff
@@ -73,7 +73,18 @@
 	 */
 	@Authorized(PrivilegeConstants.GET_OBS)
 	public Obs getObsByUuid(String uuid) throws APIException;
-	
+
+	/**
+	 * Get Revision Obs for initial Obs
+	 *
+	 * @param obs
+	 * @return obs or null
+	 * @should find revision obs for given valid obs
+	 * @should return null if no revision obs found for given obs
+	 */
+	@Authorized(PrivilegeConstants.GET_OBS)
+	public Obs getRevisionObs(Obs initialObs) throws APIException;
```

Comment:
I know our current api methods have this throws clause but it's redundant since this is a runtime exception, so going forward you don't have to include it